content/news/draft-bevy-0.14/index.md

```diff
@@ -13,7 +13,15 @@ public_draft = 1188
 
 ## What's Next?
 
-<!-- TODO What's next -->
+Sure this release was great, but what does the future hold for Bevy?
+Peering deep into the mists of time (predictions are *extra* hard when your team is almost all volunteers!), we can see some exciting work taking shape:
+
+- **Better scenes:** Scenes are one of Bevy's core building blocks: designed to be a powerful tool for saving games, authoring levels and creating reusable game objects (whether they're a form or a monster). More features and a unified syntax between assets and code should unblock progress on a UI widget abstraction, tools less boilerplates. Check out the [design doc](TODO) for more information.
```

Comment:
> (whether they're a form or a monster)

I find this phrase confusing, could you clarify what you mean? (What's a "form" in this sense?)

Reply:
Much better, thank you!